src/main/java/greencity/dto/filter/FilterDistanceDto.java

```diff
@@ -0,0 +1,24 @@
+package greencity.dto.filter;
+
+import static greencity.constant.ValidationConstants.*;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class FilterDistanceDto {
+    @Min(value = -90, message = LAT_MIN_VALIDATION)
+    @Max(value = 90, message = LAT_MAX_VALIDATION)
+    private Double lat;
+    @Min(value = -180, message = LNG_MIN_VALIDATION)
```

Comment:
is that distance will be fixed all the time or it might vary in the future? if so its better to have constants for those values so to use it both in validation and in messages combining